functions/util/events.ts

```diff
@@ -0,0 +1,17 @@
+
+import { Timestamp, getFirestore } from "firebase-admin/firestore";
+import { docToEventRaw } from "../../webapp/src/store/converters";
+
+export const getNextEvent = async () => {
+    const docRef = await getFirestore()
+    .collection("events")
+    .where("end_datetime", ">", Timestamp.now())
+    .orderBy("start_datetime", "asc");
+
+    const snapshot = await docRef.get();
+
+    const eventDoc = snapshot.docs[0]?.data();
```

Comment:
So my understanding here is that, if snapshot.docs[0] is null, docToEventRaw will return null, and therefore the resultant text displayed at the end of all of this will be "Stay Tuned!"

Reply:
Yupyup!